### PR TITLE
Update ci.yml

### PR DESCRIPTION
**Pull request by mre**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -38,6 +38,14 @@ jobs:
           chmod +x bin/tinysearch
           echo "$(pwd)/bin" >> ${GITHUB_PATH}
 
+      - name: ⏬ Install cavif
+        run: |
+          set -e
+          mkdir cavif
+          curl -sSL https://github.com/kornelski/cavif-rs/releases/download/v1.3.3/cavif-1.3.3.zip | tar xz --directory=cavif
+          chmod +x "$(pwd)/cavif/linux-generic/cavif"
+          echo "$(pwd)/cavif/linux-generic" >> ${GITHUB_PATH}
```
Comment (M3t0r) [inline .github/workflows/ci.yml]:
```suggestion
          sudo install "$(pwd)/cavif/linux-generic/cavif" /usr/local/bin
```

`install` copies the file to the destination, sets the access to `rwxr-xr-x` by default (`-m` flag) and can also create backups, set the owner and group, and the whole directory path until there.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -38,6 +38,14 @@ jobs:
           chmod +x bin/tinysearch
           echo "$(pwd)/bin" >> ${GITHUB_PATH}
 
+      - name: ⏬ Install cavif
+        run: |
+          set -e
+          mkdir cavif
+          curl -sSL https://github.com/kornelski/cavif-rs/releases/download/v1.3.3/cavif-1.3.3.zip | tar xz --directory=cavif
```
Comment (M3t0r) [inline .github/workflows/ci.yml]: This is a zip. You need `unzip`.

**Review by M3t0r — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (mre) [reply .github/workflows/ci.yml]: Yeah, I realized that, too. It was late when I did that. 😆

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mre) [reply .github/workflows/ci.yml]: TIL about `install`. Thanks!

Comment (mre) [reply .github/workflows/ci.yml]: Also, lol: https://blog.djmnet.org/2013/01/14/from-novice-to-master-and-back-again/
That's by the author of GNU `install`.

**Pull request merged**